bin/oscapd-cli

```diff
@@ -332,13 +403,39 @@ def confirm(prompt):
     sys.stderr.write("Operation aborted.\n")
     return False
 
-
-
 def main():
     parser = argparse.ArgumentParser(
         description="OpenSCAP-Daemon command line interface."
     )
     subparsers = parser.add_subparsers()
+    task_accessor = Task_accessor()
+
+    dbus_iface = None
+    try:
+        dbus_iface = get_dbus_interface()
```

Comment:
Moving `dbus_iface` here changes behavior. Previously it was possible to browse CLI help without the daemon running, this is no longer possible.

If you could avoid breaking this use-case I would really appreciate it.